src/main/java/com/rcv/GuiConfigController.java

```diff
@@ -386,6 +417,19 @@ public LocalDate fromString(String string) {
         .addListener(new TextFieldListenerNonNegInt(textFieldMinimumVoteThreshold));
 
     setDefaultValues();
+
+    try {
+      emptyConfigString =
+          new ObjectMapper()
+              .writer()
+              .withDefaultPrettyPrinter()
+              .writeValueAsString(createRawContestConfig());
+    } catch (JsonProcessingException exception) {
+      Logger.log(
+          Level.WARNING,
+          "Unable to set emptyConfigString, but everything should work fine anyway!\n%s",
```

Comment:
If that's the case, then what's the purpose of doing it? I guess if this fails, the worst that can happen is we ask if the user wants to save even though the config is empty?

Reply:
Yes, that's the worst that could happen at the moment. Kind of wanted to keep these warnings in here so we could tell if something is breaking. I don't anticipate these will get triggered much at all anyway, so it shouldn't be too confusing to the user.